From 86a298f41546dc1b2bbf3a80c1c5cd6aabf72994 Mon Sep 17 00:00:00 2001
From: Dustin Falgout <dustin@falgout.us>
Date: Tue, 21 Nov 2017 20:42:44 -0600
Subject: [PATCH] Don't update files databases when checking for updates
 because files data is not needed to determine if updates are available and
 downloading the files data takes considerably more time than the package
 databases.

---
 src/user_daemon.vala | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/user_daemon.vala b/src/user_daemon.vala
index a0f8aa2..bd32690 100644
--- a/src/user_daemon.vala
+++ b/src/user_daemon.vala
@@ -1197,14 +1197,6 @@ namespace Pamac {
 				db.update (0);
 				syncdbs.next ();
 			}
-			// refresh file dbs
-			var tmp_files_handle = alpm_config.get_handle (true, true);
-			syncdbs = tmp_files_handle.syncdbs;
-			while (syncdbs != null) {
-				unowned Alpm.DB db = syncdbs.data;
-				db.update (0);
-				syncdbs.next ();
-			}
 			string[] local_pkgs = {};
 			unowned Alpm.List<unowned Alpm.Package> pkgcache = tmp_handle.localdb.pkgcache;
 			while (pkgcache != null) {
